packages/gateway-edge-nextjs/src/cal/cal.ts

```diff
@@ -0,0 +1,150 @@
+import { type NextApiRequest, type NextApiResponse } from "next"
+import ical, { type ICalEventData } from "ical-generator"
+import { type Event } from "@dotkomonline/types"
+import { createServerSideHelpers } from "@trpc/react-query/server"
+import { appRouter, createContextInner, transformer } from "@dotkomonline/gateway-trpc"
+import { getServerSession } from "next-auth";
+import { authOptions } from "../../../auth/src/web.app"
+import jwt from 'jsonwebtoken';
+
+const helpers = createServerSideHelpers({
+  router: appRouter,
+  ctx: await createContextInner({
+    auth: null,
+  }),
+  transformer, // optional - adds superjson serialization
+})
+
+function eventUrl(event: Pick<Event, "id">) {
+  // a better to to get/configure the url?
+  return `https://dev.web.online.ntnu.no/events/${event.id}`
```

Comment:
This should be fixed before merge. @junlarsen what is the next-equivalent of Django's `reverse`

Reply:
i updated it, please re-review